core/templates/pages/topic-viewer-page/practice-tab/practice-tab.component.ts

```diff
@@ -143,6 +160,7 @@ export class PracticeTabComponent implements OnInit {
         stringified_subtopic_ids: JSON.stringify(selectedSubtopicIds)
       });
     this.windowRef.nativeWindow.location.href = practiceSessionsUrl;
+    this.loaderService.showLoadingScreen('Loading');
```

Comment:
Not sure if this is required since we are navigating away from the page in the previous line.

Reply:
Ah never mind this, just saw the discussion https://github.com/oppia/oppia/pull/15165#issuecomment-1073972994.